### ISSUE TITLE
Accessible footnotes

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_publications#256]
Request from  @LizziCBFC 

 We installed footnotes last month for use on publications but have noticed the footnote links get flagged as an accessibility failure on 'Are these links identical?'

**Comment by msayoung** [comment]
Conversation on Slack: https://localgovdrupal.slack.com/archives/C045Z40KGH0/p1750666940991019 

Lizzi Cox (Bracknell Forest)
 [Jun 23rd at 9:22 AM]
Morning. We installed footnotes last month for use on publications but have noticed the footnote links get flagged as an accessibility failure on 'Are these links identical?'


Dan Champion (Rohallion)
Is there a public URL where we can take a look please? Duplicated links aren’t necessarily an automatic accessibility failure, it depends on context.


Lizzi Cox (Bracknell Forest)
Yes, sorry should have added that! Here: [https://www.bracknell-forest.gov.uk/climate-change/climate-change-strategy-2025-2030/clima[…]strategy-2025-2030-html/climate-change-strategy-context](https://www.bracknell-forest.gov.uk/climate-change/climate-change-strategy-2025-2030/climate-change-strategy-2025-2030-html/climate-change-strategy-context)


Dan Champion (Rohallion)
Those links have the same link text but different destinations, so are a potential problem. They need to have different accessible names, I’m on mobile just now but will get a better look later - aria-label probably needed on each link if it’s not already there.



Dan Champion (Rohallion)
@Lizzi Cox (Bracknell Forest) This is an issue from the footnote module.
I’ve raised an issue there and proposed a fix, which will mitigate the issue somewhat - it’s still probably not ideal for accessibility but an improvement at least: https://www.drupal.org/project/footnotes/issues/3531828 (edited) 


Lizzi Cox (Bracknell Forest)
Amazing, thanks. Let's see if it gets picked up. :slightly_smiling_face:


Dan Champion (Rohallion)
You can fix this now on your own site, since it’s just a single Twig template file update - add the updated template file to your theme and you’ll be sorted.
